src/components/atoms/interactiveAtom.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, ReactElement, createElement as h } from 'react';
+import { css, jsx as styledH, SerializedStyles } from '@emotion/core';
+import { neutral } from '@guardian/src-foundations/palette';
+import { Format } from '@guardian/types/Format';
+import { getPillarStyles, PillarStyles } from 'pillarStyles';
+
+
+export interface InteractiveAtomProps {
+    html: string;
+    styles: string;
+    js?: string;
+    format: Format;
+}
+
+const InteractiveAtomStyles = (pillarStyles: PillarStyles): SerializedStyles => css`
+    margin: 0;
+
+    a {
+        color: ${pillarStyles.kicker};
+        text-decoration: none;
+        border-bottom: 0.0625rem solid ${neutral[86]};
+    }
+`;
+
+const InteractiveAtom: FC<InteractiveAtomProps> = (props: InteractiveAtomProps): ReactElement => {
+    const { html, styles, js, format } = props;
+    const pillarStyles = getPillarStyles(format.pillar);
+    const style = h('style', { dangerouslySetInnerHTML: { __html: styles } });
+    const script = js ? h('script', { dangerouslySetInnerHTML: { __html: js } }) : null;
+    const markup = styledH('figure', { css: InteractiveAtomStyles(pillarStyles), dangerouslySetInnerHTML: { __html: html } });
+    return <>
+        {style}
+        {script}
+        {markup}
```

Comment:
In case the script relies on an element being present?

```suggestion
        {markup}
        {script}
```

Reply:
good spot! although I think I did this for backwards compatibility. See the styles targeting h2's after script tags...

<img width="853" alt="Screen Shot 2020-05-07 at 16 03 11" src="https://user-images.githubusercontent.com/11618797/81310742-6b5fb400-907c-11ea-96e0-037387ea22c5.png">